projects/dashboards/src/test/dashboard-verification.ts

```diff
@@ -0,0 +1,32 @@
+import { NavigationService, TimeRangeService } from '@hypertrace/common';
+import { MetadataService } from '@hypertrace/distributed-tracing';
+import { GraphQlRequestService } from '@hypertrace/graphql-client';
+import { ModelJson } from '@hypertrace/hyperdash';
+import { DashboardManagerService } from '@hypertrace/hyperdash-angular';
+import { mockProvider, Spectator } from '@ngneat/spectator/jest';
+import { of } from 'rxjs';
+
+export const isValidDashboard = (
```

Comment:
How is this being used? May be we should rename this to `isValidModelJson` because it seems that's what we are testing?

Reply:
It's being used on the page level dashboard tests (didn't update any in this app though). I could go either way on the name - it's not quite either. isValidModeJson doesn't feel quite right because valid json might not produce a valid dashboard. I'll update though.